.travis.yml

```diff
@@ -1,16 +1,28 @@
 language: erlang
 otp_release:
-  - 19.3
+- 19.3
 install:
-  - make
-  - ./rebar3 update
+- make
+- "./rebar3 update"
 script:
-  - make test
-  - make xref
-  - make dialyzer
-  - make lint
+- ./rebar3 ct --suite test/plumtree_SUITE --group hyparview --case broadcast_high_client_test
+# - make xref
```

Comment:
ah sorry, i should have mentioned it, this is to try and catch logs for this failed test, doesn't happen always